recipes/deepbgc/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "deepbgc" %}
+{% set version = "0.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "57946da7fe55b0a82b1d9b8737769db62c4fad4bc6ca75766442d36f63086d2d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - deepbgc = deepbgc.main:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
+    - setuptools
+    - argparse
+    - biopython>=1.70 # support for structured comments from version 1.70
+    - scikit-learn==0.18.2 # needed for antiSMASH compatibility
```

Comment:
You don't need to pin to a particular `scikit-learn`. If you want to install with antismash and it requires a particular version then `conda create -n foo antismash deepbgc` will install an appropriate version. By not pinning it you keep compatibility with any future changes in antismash's dependencies (as well as dependencies of other tools).